### PR TITLE
Fix Blue Archive

### PR DESCRIPTION
**Pull request by piquark6046**
### To avoid invalid pull requests, please check and confirm following terms

- [X] This is not an ad/bug report
- [X] My code follows [syntax](https://adguard-dns.io/kb/general/dns-filtering-syntax/) of this project
- [X] I have performed a self-review of my own changes
- [X] My changes do not break web sites, apps and files structure

### If the problem does not fall under any category that is listed here, please write a comment below in corresponding section

- [ ] Missed ads
- [X] Website or app doesn't work properly
- [ ] Missed analytics or tracker
- [ ] Filters maintenance

### Enter the issue address

 - https://github.com/AdguardTeam/AdGuardSDNSFilter/issues/1350

### Add your comment and screenshots

I found new analytics domain (`jp.livelog.nexon.com`) in the game. 

### Terms

- [X] By submitting this issue, I agree that pull request does not contain private info and all conditions are met

### CODE REVIEW

**Comment by piquark6046** [comment]
Screenshot from 2023-05-19 07-17-27

File: Filters/exceptions.txt [hunk 1]
```diff
@@ -1,3 +1,5 @@
+! https://github.com/AdguardTeam/AdGuardSDNSFilter/issues/1350
+@@||log.ngsm.nexon.com^|
```
Comment (AdamWr) [inline Filters/exceptions.txt:2]: I guess that it should be moved to `exclusions.txt`, similar like it was in this case - https://github.com/AdguardTeam/AdGuardSDNSFilter/pull/1304

Comment (piquark6046) [reply Filters/exceptions.txt]: Fixed.

**Review by AdamWr — APPROVED** [review]

**Review by zloyden — APPROVED** [review]

File: Filters/exceptions.txt (hunk 1 above)
Comment (piquark6046) [reply Filters/exceptions.txt]: I did not face an incorrect blocking issue when a filter blocks it (not DNS filter).
Do you want me to recheck it?

**Comment by piquark6046** [comment]
Sorry for inconvenience!
 - https://gall.dcinside.com/mgallery/board/view/?id=projectmx&no=6321288
 - https://gall.dcinside.com/mgallery/board/view?id=projectmx&no=6325989&fcno=24005802

File: Filters/exceptions.txt (hunk 1 above)
Comment (Alex-302) [reply Filters/exceptions.txt]: Ok. No.

**Comment by MichalPetryka** [comment]
The fix seems to not work since it's missing the second URL.

**Comment by piquark6046** [comment]
@MichalPetryka Opened new PR.

**Comment by piquark6046** [comment]
Merged.

**Pull request merged**